### DISCUSSION TITLE
How do I change the cartURL for notifications to just the plain URL for notifications.

### DISCUSSION THREAD

**Discussion by ghost** [jef/streetmerchant#1314] [Help]
I've disabled auto add to cart but the notifications still have the cartURL in them. I could painfully go to ../stores/model and change the cartURL to just the URL for each model to all the stores but that's too much work. So is there any way I could do this change in env or at least in the notifications file?

**Comment by lozbinks** [comment]
@MemoryBytes did you ever figure this out?

**Reply by ghost** [reply]
Nope never did and good luck on getting your card btw. If you don’t mind which card are you looking for specifically.
